src/Planning.php

```diff
@@ -249,6 +249,66 @@ public static function getState($value)
     }
 
 
+    /**
+     * Get status icon
+     *
+     * @since 10.0.9
+     *
+     * @return string
+     */
+    public static function getStatusIcon($status)
+    {
+        $class = Planning::getStatusClass($status);
+        $color = Planning::getStatusColor($status);
+        $label = Planning::getState($status);
```

Comment:
To prevent issues if the label contains quotes
```suggestion
        $label = htmlspecialchars(Planning::getState($status), ENT_QUOTES);
```